releases/latest/kernel/helpers/build/configuration_snippets/infinispan-client-sessioncache.xml

```diff
@@ -0,0 +1,18 @@
+<server>
```

Comment:
please see my comments [here](https://github.com/OpenLiberty/ci.docker/pull/155/files#r396134814) about using variables that would enable non-k8s development.